pkg/common/helpers/ssar.go

```diff
@@ -0,0 +1,132 @@
+package helpers
+
+import (
+	"context"
+
+	authorizationv1 "k8s.io/api/authorization/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func CreateSelfSubjectAccessReviews(
+	ctx context.Context,
+	kubeClient kubernetes.Interface,
+	selfSubjectAccessReviews []authorizationv1.SelfSubjectAccessReview) (bool, *authorizationv1.SelfSubjectAccessReview, error) {
+
+	for i := range selfSubjectAccessReviews {
+		subjectAccessReview := selfSubjectAccessReviews[i]
+
+		ssar, err := kubeClient.AuthorizationV1().SelfSubjectAccessReviews().Create(ctx, &subjectAccessReview, metav1.CreateOptions{})
+		if err != nil {
+			return false, &subjectAccessReview, err
+		}
+		if !ssar.Status.Allowed {
+			return false, &subjectAccessReview, nil
+		}
+	}
+	return true, nil, nil
+}
+
+func GetBootstrapSSARs() []authorizationv1.SelfSubjectAccessReview {
+	var reviews []authorizationv1.SelfSubjectAccessReview
+	clusterResource := authorizationv1.ResourceAttributes{
+		Group:    "cluster.open-cluster-management.io",
+		Resource: "managedclusters",
+	}
```

Comment:
I think we should also add the resourceName here.

Update: should at least add a TODO here, we should add resourceName in the future.

Reply:
Added.